src/group/tests/kat_messages.rs

```diff
@@ -0,0 +1,512 @@
+use crate::{
+    group::GroupEpoch,
+    messages::ConfirmationTag,
+    messages::{Commit, GroupInfo, GroupSecrets, PublicGroupState},
+    node::Node,
+    prelude::*,
+    test_util::*,
+    utils::*,
+};
+use evercrypt::prelude::get_random_vec;
+
+use serde::{self, Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+pub struct MessagesTestVector {
+    key_package: String,  // serialized KeyPackage,
+    capabilities: String, // serialized Capabilities,
+    lifetime: String,     // serialized {uint64 not_before; uint64 not_after;},
+    ratchet_tree: String, /* serialized optional<Node> ratchet_tree<1..2^32-1>; */
+
+    group_info: String,    /* serialized GroupInfo */
+    group_secrets: String, /* serialized GroupSecrets */
+    welcome: String,       /* serialized Welcome */
+
+    public_group_state: String, /* serialized PublicGroupState */
+
+    add_proposal: String,            /* serialized Add */
+    update_proposal: String,         /* serialized Update */
+    remove_proposal: String,         /* serialized Remove */
+    pre_shared_key_proposal: String, /* serialized PreSharedKey */
+    re_init_proposal: String,        /* serialized ReInit */
+    external_init_proposal: String,  /* serialized ExternalInit */
+    app_ack_proposal: String,        /* serialized AppAck */
+
+    commit: String, /* serialized Commit */
+
+    mls_plaintext_application: String, /* serialized MLSPlaintext(ApplicationData) */
+    mls_plaintext_proposal: String,    /* serialized MLSPlaintext(Proposal(*)) */
+    mls_plaintext_commit: String,      /* serialized MLSPlaintext(Commit) */
+    mls_ciphertext: String,            /* serialized MLSCiphertext */
+}
+
+#[cfg(any(feature = "expose-test-vectors", test))]
```

Comment:
Shouldn't the entire module guarded with this and hence any further cfg guards be unnecessary in here?